### DISCUSSION TITLE
How to start chrome using current session/context so that to avoid login?

### DISCUSSION THREAD

**Discussion by GHRyunosuke** [GCuser99/SeleniumVBA#111] [Q&A] [answered]
Hello I wonder if anyone have ideas on this...

In case starting Chrome in the normal & manual way, login to some websites, the login status remains.
So, is there a way that SeleniumVBA could start a Chrome that inherits the same session or context as the outside/normal Chrome so that no need to log in again?

**Comment by 6DiegoDiego9** [comment, marked as answer]
Hi @GHRyunosuke, look [here](https://github.com/GCuser99/SeleniumVBA/discussions/74#discussioncomment-6112220)

**Reply by GHRyunosuke** [reply]
This works! thanks a lot and the problem is solved :)
